src/adapters/SensorApplication.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * CS 410 - Assignment 2 
+ * @author Daniyal Adzhiyev
+ * @version 1.0 05/28/2019
+ * 
+ * This Application uses the Adapter Software Pattern to create adapter's for multiple predefined sensor classes.
+ * Where the adapter classes implement the Sensor Application Interface, to 'adapt' the original sensor classes
+ * so that all of the adapter classes have the same methods needed to display the information in the Sensor 
+ * Application.
```

Comment:
The explanation of the program should only appear in the file with the main method. Having it in the other classes is unnecessarily redundant.